res/mac/FreenetTray/src/LICENSE

```diff
@@ -1,81 +1,45 @@
 Original code:
 
 This program is free software; you can redistribute it and/or modify
-    it under the terms of the GNU General Public License as published by
-    the Free Software Foundation; either version 2 of the License, or
-    (at your option) any later version.
+it under the terms of the GNU General Public License version 2, 
+as published by the Free Software Foundation.
```

Comment:
Thank you for your explanation, I understand your point that GPLv3 may prove to be unsuitable in the future. IANAL, but please note that even GPLv2 may also be unsuitable if we ever wish to publish this application to the Mac App store, as the [Terms of Service](http://www.apple.com/legal/internet-services/itunes/us/terms.html#APPS) clearly impose additional restrictions on the redistribution of the binaries, which is explicitly disallowed by GPLv2.

Neither do I care about the exact license used for this subproject. My main point is that in absence of a complete rewrite, we cannot just change the license terms without asking all copyright holders for their permission. That would be you, @toad and @RichyRich, according to Git history.

If we can somehow consider this (or a further derivative) a complete rewrite, you might want to indicate so in some commit message (possibly getting rid of the original work in the same commit).